### PR TITLE
Add rule for blank lines after multiline assignment

### PR DESCRIPTION
**Pull request by corroded**
This is for #60 . I made the examples as simple as I could think of and separated the examples as @eksperimental suggested.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -291,6 +291,47 @@ Translations of the guide are available in the following languages:
   end
   ```
 
+* <a name="add-blank-line-after-multiline-assignment"></a>
+  Add a blank line after a multiline assignment as a
+  visual cue that the assignment is 'over'
+  <sup>[[link](#add-blank-line-after-multiline-assignment)]</sup>
```
Comment (eksperimental) [inline README.md]: messing ending period.

File: README.md [hunk 2]
```diff
@@ -291,6 +291,47 @@ Translations of the guide are available in the following languages:
...
+  another_string <> some_string
+
+  # preferred
+  some_string =
+    "Hello"
+    |> String.downcase
+    |> String.strip
+
+  another_string <> some_string
+
```
Comment (eksperimental) [inline README.md]: break the examples into two different examples, by closing it with "```" and reopening it with "```elixir"

**Comment by corroded** [comment]
@eksperimental thanks for reviewing! Addressed the comments 👍

**Comment by christopheradams** [comment]
Great!

**Comment by eksperimental** [comment]
thank you @corroded

**Pull request merged**